app/javascript/components/maps/map/index.js

```diff
@@ -34,16 +33,21 @@ class MapContainer extends PureComponent {
       bbox,
       geostoreBbox,
       setMapSettings,
-      layerBbox,
-      setLandsatBasemap
+      layerBbox
     } = this.props;
 
     // update landsat basemap when changing zoom
-    if (basemap.value === 'landsat' && zoom !== prevProps.zoom) {
-      setLandsatBasemap({
-        basemap,
-        year: basemap.year,
-        zoom
+    if (
```

Comment:
If the zoom is now part of the url and is fetched from the service I don't think this is still needed?